pyxrf/model/setting.py

```diff
@@ -181,18 +313,10 @@ def update_roi(self, element_list, std_ratio=4):
             self.roi_dict.update({v: roi})
 
         # remove old items not included in element_list
-        for k in six.iterkeys(self.roi_dict):
+        for k in six.iterkeys(self.roi_dict.copy()):
```

Comment:
Can the use of `six` be removed here, as you touched the line anyway?

Reply:
Fixed

Reply:
Thanks.